src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -142,9 +142,16 @@
         get-br-list (fn [match-str]
                       (map #(Integer/parseInt (second %))
                            (re-seq #"\$(\d+)" match-str)))
+        get-certname-list (fn [rule]
+                            (cond
+                              (string? rule) (list rule)
+                              (map? rule) (list (get rule :certname ""))
+                              (seq? rule) (for [x rule]
```

Comment:
I don't think `seq?` works here.  If I change this to `sequential?`, though, it seems to work for the `vector` type that this can come in as from HOCON.